tests/test_dynamic_movement_primitives.py

```diff
@@ -0,0 +1,35 @@
+import conftest
+import numpy as np
+from PathPlanning.DynamicMovementPrimitives import \
+            dynamic_movement_primitives
+
+
+def test_1():
+    # test that trajectory can be learned from user-passed data
+    T = 5
+    t = np.arange(0, T, 0.01)
+    sin_t = np.sin(t)
+    train_data = np.array([t, sin_t]).T
+
+    DMP_controller = dynamic_movement_primitives.DMP(train_data, T)
+    DMP_controller.recreate_trajectory(train_data[0], train_data[-1], 4)
```

Comment:
Maybe a test to check the path result is not empty here?

Reply:
Added a test to do this and named it test 2 (old test 2 that checks if path is close to training data is now test 3)